lib/commands/context.js

```diff
@@ -0,0 +1,174 @@
+import _ from 'lodash';
+import logger from '../logger';
+import webviewHelpers from '../webview-helpers';
+import Chromedriver from 'appium-chromedriver';
+import { errors } from 'mobile-json-wire-protocol';
+import { NATIVE_WIN, WEBVIEW_BASE, WEBVIEW_WIN, CHROMIUM_WIN } from '../webview-helpers';
+
+let commands = {}, helpers = {}, extensions = {};
+
+helpers.defaultContextName = function () {
+  return NATIVE_WIN;
+};
+
+helpers.defaultWebviewName = function () {
+  return WEBVIEW_BASE + this.opts.appPackage;
+};
+
+helpers.isWebContext = function () {
+  return this.curContext !== null && this.curContext !== NATIVE_WIN;
+};
+
+function isChromedriverContext (viewName) {
+  return viewName.indexOf(WEBVIEW_WIN) !== -1 || viewName === CHROMIUM_WIN;
+}
+
+helpers.startChromedriverProxy = async function (context) {
+  logger.debug("Connecting to chrome-backed webview");
```

Comment:
It seems like `context` is readable (not a random udid or something) so it might be nice to log it, as we do in `setupExistingChromedriver`.